core/server/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -837,47 +834,49 @@ public void streamToJournalCheckpoint(JournalOutputStream outputStream) throws I
   }
 
   /**
-   * Adds the inode represented by the entry parameter into the inode tree. If the inode entry
+   * Adds the file represented by the entry parameter into the inode tree.
+   *
+   * @param entry the journal entry representing an inode
+   */
+  public void addInodeFileFromJournal(InodeFileEntry entry) {
```

Comment:
because we are simply adding the inode file to the inode tree based on journaled information (which was already checked for permissions when it was validated)

we only throw AccessControlException now for replaying adding the root inode directory when its owner does not much the current owner

more importantly, this PR does not change the behavior